comps/cores/mega/orchestrator.py

```diff
@@ -166,6 +168,7 @@ def process_outputs(self, prev_nodes: List, result_dict: Dict) -> Dict:
             all_outputs.update(result_dict[prev_node])
         return all_outputs
 
+    @opea_telemetry
```

Comment:
in this `execute` method we basically has two cases:

1) streaming responses
2) non-streaming responses

for 2) , we can use this `opea_telemetry` decorator because the time span is selected correctly
for 1),  this `opea_telemetry` decorator may not work well because the Streaming response is directly returned (at that moment, the first token is not yielded at all!) before the first yield so the time span is not the actual execution time.

Reply:
`self.metrics.request_update(req_start)` line in the code shows the place after which processing of **streaming** call ends.